runtime/syntax/shaderslang.vim

```diff
@@ -0,0 +1,377 @@
+" Vim syntax file
+" Language:     Slang
+" Maintainer:	Austin Shijo <epestr@proton.me>
+" Last Change:	2024 Jan 05
+
+" For version 5.x: Clear all syntax items
+" For version 6.x: Quit when a syntax file was already loaded
+if version < 600
+  syntax clear
+elseif exists("b:current_syntax")
+  finish
+endif
+
+" Read the C syntax to start with
+if version < 600
+  so <sfile>:p:h/c.vim
+else
```

Comment:
remove the `if ... else .. endif ` part and keep only the things inside the `else` block